integrations-and-supported-tools/bokeh/scripts/run_examples.sh

```diff
@@ -0,0 +1,3 @@
+set -e
```

Comment:
What does this line do? Does this work outside of Windows?

Reply:
`set -e`  terminates execution of the script if any command fails, as opposed to the default script behaviour of continuing execution from the next command. This works in bash scripts